en/config/safety.md

```diff
@@ -225,19 +225,12 @@ Parameter | Description
 [COM_OBL_ACT](../advanced_config/parameter_reference.md#COM_OBL_ACT) | Failsafe action if no RC is available: Land mode, Hold mode, Return mode.
 [COM_OBL_RC_ACT](../advanced_config/parameter_reference.md#COM_OBL_RC_ACT) | Failsafe action if RC is available: Position mode, Altitude mode, Manual mode, Return mode, Land mode, Hold mode.
 
+### Mission Feasibility Checks
```

Comment:
While not strictly a "safety section" it is quite closely related, and I think this is a reasonable place to cross link the information. Reasonable?

Reply:
Yes perfectly reasonable :+1: